libs/python.json

```diff
@@ -126,6 +126,7 @@
   {"id": "py.johny-cache", "imports": ["johny"], "name": "Johny Cache", "repo": "jmoiron/johnny-cache", "tags": ["django-caching"], "tech": ["python-web"]},
   {"id": "py.pylibmc", "imports": ["pylibmc"], "name": "pyblibmc", "repo": "lericson/pylibmc", "tags": ["memcached-client"], "tech": ["python-web"]},
   {"id": "py.test", "imports": ["pytest"], "name": "pytest", "repo": "pytest-dev/pytest", "tags": ["unit-testing"], "tech": ["testing"]},
-  {"id": "py.connexion", "imports": ["connexion"], "name": "connexion", "repo": "zalando/connexion", "tags": ["micro-framework"], "tech": ["python-web"]}
-  {"id": "py.pyramid", "imports": ["pyramid"], "name": "pyramid", "repo": "Pylons/pyramid", "tags": ["micro-framework"], "tech": ["python-web"]}
+  {"id": "py.connexion", "imports": ["connexion"], "name": "connexion", "repo": "zalando/connexion", "tags": ["micro-framework"], "tech": ["python-web"]},
+  {"id": "py.pyramid", "imports": ["pyramid"], "name": "pyramid", "repo": "Pylons/pyramid", "tags": ["micro-framework"], "tech": ["python-web"]},
+  {"id": "py.or-tools", "imports": ["or-tools"], "name": "or-tools", "repo": "google/or-tools", "tags": ["optimization"], "tech": ["Operations Research tools"], "examples": ["https://github.com/google/or-tools/blob/master/examples/python/knapsack.py"]}  
```

Comment:
would it suffice if we put it under 'computational science' tech instead? adding new tech is takes longer for us, but we will do it if you think 'operations research' should take a separate tech category.

My suggestions is to do: ```"tags": ["operations-research"], "tech": ["computational-science"]```

Reply:
Thank you for your answer.  I did the tag and tech modification as suggested, it is ok for you ?